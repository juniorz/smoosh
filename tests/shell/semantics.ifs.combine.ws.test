unset IFS 
echo                  > spaced
printf "%b" '\tx'    >> spaced
echo                 >> spaced
echo "          5"   >> spaced
printf '%b' ' 12\t ' >> spaced
echo `cat spaced`
IFS=$(printf '%b' ' \n\t')
echo `cat spaced`