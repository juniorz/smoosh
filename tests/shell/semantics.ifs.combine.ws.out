x 5 12
x 5 12
